src/h5web/toolbar/controls/ToggleBtn.tsx

```diff
@@ -30,7 +30,7 @@ function ToggleBtn(props: Props) {
     <button
       className={styles.btn}
       type="button"
-      onClick={onChange}
+      onClick={() => onToggle()}
```

Comment:
What is the difference between:
```
onClick={() => onToggle()}
```
and
```
onClick={onToggle}
``` 
?

Reply:
In the second case, `onToggle` receives the `MouseEvent` as first argument, which means that if `onToggle` declares a parameter `force` and then does something with it, like `if (!!force)`, then it will pass the condition even though what we wanted was to call `onToggle()` without passing a `force` argument.